docs/source/dev/profiling/profiling_index.rst

```diff
@@ -0,0 +1,33 @@
+Profiling vLLM 
+=================================
+
+We support tracing vLLM workers using the ``torch.profiler`` module. You can enable tracing by setting the ``VLLM_TORCH_PROFILER_DIR`` environment variable to the directory where you want to save the traces: ``VLLM_TORCH_PROFILER_DIR=/mnt/traces/``
+
+The OpenAI server also needs to be started with the ``VLLM_TORCH_PROFILER_DIR`` environment variable set.
+
+When using ``benchmarks/benchmark_serving.py``, you can enable profiling by passing the ``--profile`` flag.
```

Comment:
any reason why we just don't ask users to use env var inestaed?

Reply:
I find it easier to use flags to control rather than env vars for the benchmark. For dev, you can leave the profiler enabled for the engine but select when to trigger the profiler from cmdline